CHECKOUT.md

```diff
@@ -0,0 +1,37 @@
+# Documentation - Stripe Functions
+
+## Introduction
+
+Amplify uses Stripe to process donations for non-profit organizations. Use these API endpoints to store and complete transactions.
```

Comment:
Amplify uses Stripe to process donations for non-profit organizations.  Amplify uses these API endpoints to store and complete transactions.